engines/scc_proxy/lib/scc_proxy/engine.rb

```diff
@@ -372,6 +373,15 @@ def has_no_regcode?(auth_header)
         protected
 
         def scc_activate_product
+          if (@system.system_information &&
+              JSON.parse(@system.system_information)['cloud_provider'].casecmp('microsoft').zero? &&
```

Comment:
Should not be CSP specific and CSP could have an offer the meets the definitions that make us exclude ltss registration.

Reply:
Fair point, I understood it should be only for Microsoft or CSP specific

That means I will update the other two verification codebases